lms/djangoapps/bulk_email/migrations/0003_multiple_choices_to_option.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import util.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('bulk_email', '0002_data__load_course_email_template'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='courseemail',
+            name='to_option',
+            field=util.models.SerializedChoicesField(default=[b'myself'], max_length=64, choices=[(b'myself', b'Myself'), (b'staff', b'Staff and instructors'), (b'learners', b'All students')]),
```

Comment:
I worry about max_length only being 64 chars. We are plan to add a number of new options here (for cohort selection), and I don't think they will all fit.

Also, the new choices being added will need to be dynamic based on the cohorts in existence at the time. See https://openedx.atlassian.net/browse/TNL-4357.